src/assets/scss/layout/layout.scss

```diff
@@ -51,6 +51,16 @@
   min-width: 190px;
 }
 
+.title {
+  background: none;
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
+	cursor: pointer;
+	outline: inherit;
```

Comment:
Fix alignment

Reply:
The alignment is actually okay here (you can see it when you open it in a text editor). Not sure why the indentation is weird on the web UI.